### ISSUE TITLE
[FF 66.03] Drag and drop to delete doesn't function 

### ISSUE THREAD

**Issue by socketbox** [ReDEnergy/SessionSync#69]
I'm using Session-Sync on the Linux version of FF 66.03. Everything works nicely, aside from the "drag-to-trash" delete feature. When I click and begin dragging an item, the trash can appears in the corner. As I pass over it with the tab I'm dragging, it grows larger and a tooltip appears. But when I release it, nothing happens. The trash can remains enlarged while my mouse is over it, the tooltip is still there, but the tab snaps back to the list of open tabs and nothing is removed.

**Comment by ReDEnergy** [comment]
Yeah... it's not actually a bug, but mostly a missing feature (half implemented by mistake) because open tabs use the same underlying UI structure elements similar to the bookmarks. So basically I simply never though about adding that feature, not sure how I missed it but it's definitely something that must be available. I will try to push an update as soon as possible to "fix it". Thanks for point it out.
